categories.xml

```diff
@@ -1,4 +1,5 @@
 <categories>
-	<category name="Uncategorized" slug="uncategorized"/>
+	<category name="Uncategorized" slug="uncategorized"></category>
+        <category name="Configuration and Utilities" slug="config"></category>
```

Comment:
It's up to @rxaviers, but shouldn't there be two categories here: `Configuration` and `Utilities`? Then it's also perfectly possible to add two categories on one article.

Also, I know it's nitpicky, but could you please fix the indentation in all these files? You used spaces while we use tabs.